From 5afccffeaf08b0c4444e91e1f49de13da7a4079b Mon Sep 17 00:00:00 2001
From: Rohith Sharma K S <rohithsharmaks@apache.org>
Date: Thu, 27 Oct 2016 11:57:17 +0530
Subject: [PATCH 2152/2848] YARN-4363. In TestFairScheduler, testcase should
 not create FairScheduler redundantly.
 Conntributed by Tao Jie.

(cherry picked from commit e29cba61a0bec0629de287f67ae6eed526295ffb)

Change-Id: I8271c74e6ca913ff6ac6fd27bb16de8f3102b076
---
 .../scheduler/fair/TestFairScheduler.java          |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestFairScheduler.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestFairScheduler.java
index 3c5ee26..80a8da1 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestFairScheduler.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestFairScheduler.java
@@ -164,7 +164,6 @@ public void tearDown() {
 
   @Test (timeout = 30000)
   public void testConfValidation() throws Exception {
-    scheduler = new FairScheduler();
     Configuration conf = new YarnConfiguration();
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_MB, 2048);
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MAXIMUM_ALLOCATION_MB, 1024);
@@ -236,7 +235,6 @@ public void testLoadConfigurationOnInitialize() throws IOException {
   
   @Test  
   public void testNonMinZeroResourcesSettings() throws IOException {
-    scheduler = new FairScheduler();
     YarnConfiguration conf = new YarnConfiguration();
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_MB, 256);
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_VCORES, 1);
@@ -254,7 +252,6 @@ public void testNonMinZeroResourcesSettings() throws IOException {
   
   @Test  
   public void testMinZeroResourcesSettings() throws IOException {  
-    scheduler = new FairScheduler();
     YarnConfiguration conf = new YarnConfiguration();
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_MB, 0);
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_VCORES, 0);
-- 
1.7.9.5

